crew

```diff
@@ -51,6 +52,16 @@ require_relative 'lib/docopt'
 begin
   args = Docopt::docopt(DOC)
 rescue Docopt::Exit => e
+  puts "Could not understand \"crew #{ARGV.join(' ')}\".".lightred
+  cmds = ["build", "download", "files", "help", "install", "remove", "search", "update", "upgrade", "whatprovides"]
```

Comment:
I'm wondering if we can use this array generically so we only need to manage all the commands in one place.  It might even be a good idea to store it in the config directory like `CREW_CONFIG_PATH + 'device.json'` with all the other metadata like each help command message strings and even the docopt usage section.  It would require some serious refactoring, however.

Reply:
Yes, that would be good. While that isn't implemented, I think this is the simplest solution though :P

Reply:
Agreed but we should keep this thought in the back of our minds moving forward.